docs(ai): smart labeling - MTA-1571 (#210)
scaleway/docs-content: ai/smart-labeling/how-to/create-labeling-workspace.mdx

---
title: How to create a labeling workspace
description: This page explains how to create a labeling workspace
dates:
  validation: 2021-11-05
  posted: 2021-11-05
---

<Message type="requirement">

> - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)  

Reviewer: Another requirement is that they have created an object storage bucket. Maybe we could also add a few lines of introduction, explaining the principle of creating an object storage bucket and uploading the images and videos to be labelled (with a link to the relevant how tos for creating buckets / uploading files) , and then creating a labeling workspace and associating the bucket with it. I believe that it is imperative that hte bucket be created BEFORE the workspace, else they have to delete the workspace and start all over again.

</Message>

> 1. Click **Smart Labeling** in the **AI** section of the side menu. The Smart Labeling splash screen displays:

Reviewer: ```suggestion
1. Click **Smart Labeling** in the **AI** section of the side menu. The Smart Labeling creation page displays:
```

> <Lightbox src="scaleway-smart-labeling-splash.png" />

Reviewer 2: indent here

2. Click **Create a labeling workspace**. The workspace creation wizard displays.

3. Enter the required information for your labeling worksapce:

>     * Enter a Name and a description for the workspace.

Reviewer: ```suggestion
    * Enter a name and, optionally, a description for the workspace.
```

    * Select the Object Storage bucket you want to use with your workspace.
    * Enter a username for the admin user.
    * Enter the email address of the admin user. 
    * Enter a secret password for the admin user.

4. Confirm by clicking **Create a labeling workspace**. Your workspace will be created.

> Once ready the workspace displaces in the list of your Smart Labeling workspaces.

Reviewer 2: ```suggestion
Once ready, the workspace displays in the list of your Smart Labeling workspaces.
```

description: This page explains how to create a labeling workspace
dates:
  validation: 2021-11-05
  posted: 2021-11-05
---

<Message type="requirement">

  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com) 
  - You have [created an Object Storage bucket](/storage/object/quickstart/)

</Message>

1. Click **Smart Labeling** in the **AI** section of the side menu. The Smart Labeling creation page displays:

  <Lightbox src="scaleway-smart-labeling-splash.png" alt="" />

2. Click **Create a labeling workspace**. The workspace creation wizard displays.

3. Enter the required information for your labeling worksapce:
    * Enter a name and, optionally, a description for the workspace.
    * Select the Object Storage bucket you want to use with your workspace.
    * Enter a username for the admin user.
    * Enter the email address of the admin user. 
    * Enter a secret password for the admin user.

4. Confirm by clicking **Create a labeling workspace**. Your workspace will be created.

  Once ready, the workspace displays in the list of your Smart Labeling workspaces.

<Navigation title="See Also">

>   <NextButton to="/ai/smart-labeling/how-to/access-annotation-server">How to access the annotation server</NextButton>

Reviewer 2: ```suggestion
  <NextButton to="/ai/smart-labeling/how-to/access-annotation-server/">How to access the annotation server</NextButton>
```
